cypress/integration/pages/FIX/index.js

```diff
@@ -0,0 +1,31 @@
+import runTestsForPage from '../../../support/helpers/runTestsForPage';
```

Comment:
Not sure about the page name 'FIX' but it matches the pageType that already exists for feature index - I believe this comes from how the page is defined in the metadata from the ares response.